pkgs/by-name/ut/util-linux/package.nix

```diff
@@ -48,6 +48,8 @@ stdenv.mkDerivation rec {
       ./rtcwake-search-PATH-for-shutdown.patch
       # https://github.com/util-linux/util-linux/pull/3013
       ./fix-darwin-build.patch
+      # https://github.com/util-linux/util-linux/pull/3479 (fixes https://github.com/util-linux/util-linux/issues/3474)
+      ./fix-mount-regression.patch
```

Comment:
We should use fetchpatch with the patch URL. Just put .patch after the URL.

Reply:
Can't do that in util-linux. It's an infinite recursion.

Reply:
oof, good call